### ISSUE TITLE
3dconv video cls problem

### ISSUE THREAD

**Issue by ZHEQIUSHUI** [dmlc/gluon-cv#1124]
when i train my custom data(nclass is 2),no matter how mang epochs i have training.the acc just about 50%(i think it just classify only one class?)

thank you for you read if you have free time!

this is my option(it is so long....):
>accumulate=1, 
batch_norm=False, 
batch_size=5,
clip_grad=0, 
crop_ratio=0.875, 
data_dir='', 
dataset='custom', 
dtype='float32', 
eval=False, 
fast_temporal_stride=2, 
hard_weight=0.5, 
hashtag='', 
input_5d=False, 
input_size=224, 
kvstore='device', 
label_smoothing=False, 
last_gamma=False, 
log_interval=50, 
logging_file='train.log', 
lr=0.001, 
lr_decay=0.1, 
lr_decay_epoch='5,15', 
lr_decay_period=0, 
lr_mode='step', 
mixup=False, 
mixup_alpha=0.2, 
mixup_off_epoch=0, 
mode=None, 
model='i3d_resnet50_v1_custom', 
momentum=0.9, 
new_height=0, 
new_length=16, 
new_step=1,
new_width=0, 
no_wd=False, 
num_classes=2,
num_crop=1, 
num_epochs=20, 
num_gpus=1, 
num_segments=1, 
num_workers=4, 
partial_bn=False, 
prefetch_ratio=2.0, 
resume_epoch=0, 
resume_params='', 
resume_states='', 
save_dir='params', 
save_frequency=10, 
scale_ratios='1.0, 0.875, 0.75, 0.66', 
slow_temporal_stride=16, 
slowfast=False, 
teacher=None, 
temperature=20, 
train_list='../dataset/train.txt', 
use_amp=False, 
use_decord=False, 
use_gn=False, 
use_pretrained=True, 
use_se=False, 
use_tsn=False, 
val_data_dir='', 
val_list='../dataset/val.txt', 
video_loader=True, 
warmup_epochs=0, 
warmup_lr=0.0, 
wd=0.0001

this is my train log:
>[Epoch 000] training: accuracy=53.061224	 loss=11.034944
[Epoch 000] speed: 13 samples/sec	time cost: 18.987665
[Epoch 000] validation: acc-top1=68.000000 acc-top5=100.000000 loss=18.733686

>[Epoch 001] training: accuracy=55.510204	 loss=23.471720
[Epoch 001] speed: 22 samples/sec	time cost: 11.574985
[Epoch 001] validation: acc-top1=44.000000 acc-top5=100.000000 loss=103.520424

>[Epoch 002] training: accuracy=49.795918	 loss=10.128425
[Epoch 002] speed: 22 samples/sec	time cost: 11.568123
[Epoch 002] validation: acc-top1=52.000000 acc-top5=100.000000 loss=1.192754

>[Epoch 003] training: accuracy=49.387755	 loss=7.910957
[Epoch 003] speed: 22 samples/sec	time cost: 11.624424
[Epoch 003] validation: acc-top1=52.000000 acc-top5=100.000000 loss=0.702443

>Epoch[004] Batch [0049]/[0050]	Speed: 22.814440 samples/sec	 accuracy=51.600000	 loss=8.536176	 lr=0.000100
[Epoch 004] training: accuracy=51.600000	 loss=8.536176
[Epoch 004] speed: 22 samples/sec	time cost: 11.848721
[Epoch 004] validation: acc-top1=56.000000 acc-top5=100.000000 loss=23.990627

>[Epoch 005] training: accuracy=46.530612	 loss=11.483197
[Epoch 005] speed: 22 samples/sec	time cost: 11.690950
[Epoch 005] validation: acc-top1=40.000000 acc-top5=100.000000 loss=7.925127

>[Epoch 006] training: accuracy=57.142857	 loss=5.142479
[Epoch 006] speed: 21 samples/sec	time cost: 11.964715
[Epoch 006] validation: acc-top1=56.000000 acc-top5=100.000000 loss=0.945472

**Comment by ZHEQIUSHUI** [comment]
i train in  "fineturn_i3d_custm.py" is normal,and the acc is fast to be high.but "train_recognizer.py" have many additional option,but i don`t known what is it mean.....i am so sorry for my ignorance

**Comment by ZHEQIUSHUI** [comment]
i have solve this,it is cause i have use "pretrained",when i set "pretrained=False",the traing is normal!
